nc/source/graphics/techniques/TechniqueManager.h

```diff
@@ -2,6 +2,7 @@
 
 #include "graphics/techniques/Technique.h"
 #include "graphics/techniques/PhongShadingTechnique.h"
+#include "graphics/techniques/WireframeTechnique.h"
```

Comment:
It looks like this doesn't need PhongShadingTechnique.h or WireframeTechnique.h.